spec/dummy/spec/system/integration_spec.rb

```diff
@@ -66,10 +66,10 @@ def finished_all_ajax_requests?
 
     context "React Hooks" do
       context "Simple stateless component" do
-        include_examples "React Component", "div#HelloWorldApp-react-component-6"
+        include_examples "React Component", "div#HelloWorld-react-component-6"
       end
       context "render function that takes props" do
-        include_examples "React Component", "div#HelloWorldApp-react-component-7"
+        include_examples "React Component", "div#HelloWorld-react-component-7"
```

Comment:
An idea on why this changed?

Reply:
Those are the Ids that you used in your initial commit: https://github.com/shakacode/react_on_rails/pull/1268/commits/be37b3410de1974f16066951bc41fa73fa6f3c61?file-filters%5B%5D=.erb#diff-13675373e6eaa2b6e91995c3ec8974f7